scripts/getPVAliases.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/bash
+
+# define usage
+usage()
+{
+cat << EOF
+usage: gatherPVAliases [-h] [-d] patt hutch
+
+gathers all record <-> alias associations from a child's ioc.cfg, st.cmd, and
+parent ioc.cfg.
+
+positional arguments:
+  patt
+  hutch
+
+options:
+  -h, --help     show this help message and exit
+  -d, --dry_run  Forces a dry run for the script. No files are saved.       
+
+EOF
+}
+
+# catch improper number of args or asking for help
```

Comment:
I believe you can just let python's argparse handle bad args and the help request, if you pass all the arguments through to your python script.

Reply:
`source pcds_conda` is greedy and eats `-h` :(

So I tossed this in there as a hacky work-around

Reply:
Fixed with suggested changes